### PR TITLE
Angular 10 update

### PR DESCRIPTION
**Pull request by hebelt**
#109 updated to angular 10

### CODE REVIEW

**Comment by fcarrero** [comment]
any news about this pull request? I'm updating my project to angular 10, but I have a couple of problems

**Comment by seanlawrenz** [comment]
This update looks good. I'd like to see it be branched as a version 10.

**Comment by viniciosarodrigues** [comment]
I'm just waiting for this PR to perform the migration of my application.

**Comment by MattWeirich** [comment]
Also waiting on this to update an application to Angular 10, Any updates?

**Comment by AchillesKarki** [comment]
Waiting for this merge as well. Any updates?

**Comment by joeylgutierrez** [comment]
@Zak-C you still around?

**Comment by hkarakose** [comment]
Is this update merged? How can we use this update @Zak-C ?

**Comment by hakimio** [comment]
@marklb has published a version with this PR to npm: [@marklb/ngx-loading](https://www.npmjs.com/package/@marklb/ngx-loading)
Thank you @hebelt and @marklb

File: e2e/tsconfig.e2e.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../tsconfig.json",
+  "extends": "../tsconfig.base.json",
```
Comment (andredesousa) [inline e2e/tsconfig.e2e.json:2]: This file was deprecated in Angular 10.1. More details:

https://github.com/angular/angular-cli/pull/18478

**Review by andredesousa — COMMENTED** [review]
I suggest closing this PR in favor of:

https://github.com/Zak-C/ngx-loading/pull/118

This PR has old Angular version and the other does not.

**Pull request closed without merging**